packages/react/tabs/src/Tabs.stories.tsx

```diff
@@ -312,17 +409,25 @@ const RECOMMENDED_CSS__TABS__CONTENT = {
 
 const contentClass = css({
   ...RECOMMENDED_CSS__TABS__CONTENT,
-
   padding: '1em',
-  border: '1px solid #eee',
```

Comment:
oh that would be why it's shifted no? 🙂

Reply:
https://github.com/radix-ui/primitives/pull/1346#discussion_r878163522 yes, I forgot I did that haha